hydrolib/core/basemodel.py

```diff
@@ -441,6 +469,38 @@ def is_empty(self) -> bool:
         """
         return not any(self._cache_dict)
```

Comment:
This should also work: `return not self._cache_dict`. Knowing that, this function may not be needed since the statement is so simple.

Reply:
i think this function add for readability wise.

Reply:
Slightly unconventional, but personal preference in the end ^^